test/random.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>QUnit Randomization Test Suite</title>
+    <link rel="stylesheet" href="../dist/qunit.css">
+    <script src="../dist/qunit.js"></script>
+    <script>
+      QUnit.config.seed = "7x9";
```

Comment:
I can do that, but we have other tests following this pattern as well `logs.html` and `string-filter.html` for example. Should we update those in a separate PR?